package.json

```diff
@@ -54,7 +54,8 @@
   "types": "dist/index.d.ts",
   "scripts": {
     "lint": "eslint ./package ./tests",
```

Comment:
`yarn lint` is executed by the GitHub action, this way is controlled automatically in the examples PRs.

```suggestion
    "lint": "eslint ./package ./tests ./examples",
```